### DISCUSSION TITLE
Missing ports Ravenports for a possible migration

### DISCUSSION THREAD

**Discussion by tuxillo** [DragonFlyBSD/DragonFlyBSD#26] [General]
What ports is Ravenports missing compared to DPorts?

Ravenports catalog: http://www.ravenports.com/catalog/
Deltaports status files: https://github.com/DragonFlyBSD/DeltaPorts/tree/master/ports

**Comment by mneumann** [comment]
Required:

- [ ] www/h2o
- [ ] lang/elixir
- [ ] devel/esbuild
- [ ] mail/opensmtpd-filter-dkimsign
- [ ] mail/opensmtpd-filter-senderscore
- [ ] www/chromium
- [ ] graphics/darktable
- [ ] graphics/rawtherapee
- [ ] x11-wm/hikari
- [ ] textproc/ripgrep
- [ ] editors/helix

Nice to have:

- [ ] lang/ghc
- [ ] finance/hs-ledger
